libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -52,6 +52,11 @@ const readingListReducer = createReducer(
   ),
   on(ReadingListActions.failedAddToReadingList, (state, action) =>
   readingListAdapter.removeOne( action.book.id, state)
+),
+on(ReadingListActions.markAsFinished, (state, action) =>
+
+readingListAdapter.updateOne({id: action.item.bookId, changes:{finished: true, finishedDate: new Date().toISOString()}}, state)
```

Comment:
Again, this line is too long to be readable... I would use Prettier to take care of these things, or you can just add the line changes where needed. I know there's controversy around the 80-column rule, but having these long lines with nested objects only makes the code less readble... :(